docs/source/install.rst

```diff
@@ -1,265 +1,711 @@
+-----------------------------
 
-- ``geopm-service``: Sets up and enables the geopm systemd service.
-- ``libgeopmd2``: Provides the library that supports the PlatformIO interface.
-- ``python3-geopmdpy``: Houses the implementation of geopmd, CLI utilities, and bindings for PlatformIO.
-- ``geopm-service-devel``: Provides headers and man pages for C and C++ APIs supportable by ``libgeopmd2``.
-- ``geopm-doc``: Provides all GEOPM man pages.
+Install packages created by GEOPM development team to install the latest stable
+release of GEOPM.
 
-v3.0.1
-------
 
-- ``geopm-service``: Sets up and enables the geopm systemd service.
-- ``libgeopmd2``: Provides the library that supports the PlatformIO interface.
-- ``python3-geopmdpy``: Houses the implementation of geopmd, CLI utilities, and bindings for PlatformIO.
-- ``geopm-service-devel``: Provides headers and man pages for C and C++ APIs supportable by ``libgeopmd2``.
+GEOPM Access Service - Official
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Installation of ``geopm-service`` will involve the dependency packages:
-``libgeopmd2``, ``python3-geopmdpy`` and ``python3-dasbus``. However, explicit
-installation is obligatory for anyone seeking to use ``geopm-service-devel`` or
-the docs packages.
+Instructions on how to install the latest official release of the GEOPM Access
+Service and its dependency packages is broken down by OS and OS version.
 
-----
+.. tabs::
 
-GPU Support
------------
+   .. group-tab:: openSUSE  
 
-In order to leverage the GEOPM Service's support for GPUs, you must install the
-necessary libraries for either Intel GPUs (levelzero) or Nvidia GPUs (NVML).
+      .. tabs::
 
-Prerequisites for Level Zero
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+         .. group-tab:: 15.3
+
+            .. tabs::
 
-``level-zero`` allows GEOPM to read metrics and enact controls from Intel GPUs.
-Currently, none of the Linux distributions supporting GEOPM provide this
-package upstream.
+               .. group-tab:: Limited GPU support
 
-To install ``level-zero``, use the one of the following:
+                  .. code-block:: bash
 
-For up-to-date distributions (e.g. SLES 15.4 and Ubuntu 22.04):
+                     # Add zypper repo
+                     zypper addrepo https://download.opensuse.org/repositories/hardware/SLE_15_SP3_Backports/hardware.repo
+                     zypper refresh
+                     # GEOPM Access Service python module
+                     zypper install python3-geopmdpy
+                     # SystemD service configuration and geopmd executable
+                     zypper install geopmd
+                     # C/C++ development files
+                     zypper install libgeopmd-devel
+                     # Man pages for GEOPM Access Service
+                     zypper install geopmd-doc
+                     # Man pages for C/C++ development
+                     zypper install libgeopmd-doc
+
+
+               .. group-tab:: Intel GPU support
```

Comment:
Does "limited" just mean DRM-only? Maybe explain in the paragraph above the tab groups. Also what about NVML builds? e.g.,

> GEOPM works with Intel GPUs when GEOPM is built with LevelZero support and NVIDIA GPUs when GEOPM is built with NVML support. Additional limited support for GPUs exists when the GPUs are visible in a Linux Direct Rendering Manager (DRM) sysfs interface.